contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -14,6 +14,7 @@ contract TrueCreditAgency is UpgradeableClaimable {
 
     uint8 constant MAX_CREDIT_SCORE = 255;
     uint256 constant MAX_RATE_CAP = 50000;
+    uint256 constant REPAYMENT_PERIOD = 31 days;
```

Comment:
Not sure if it wouldn't be nice if this was configurable.

Reply:
right